umpire/deploy.py

```diff
@@ -289,8 +288,6 @@ def __remove_and_deploy_to_destination__(self, fetcher, entry, destination_file)
                 path.symlink(entry, destination_file)
             elif os.path.isdir(entry):
                 logger.debug("Copying with tree copy")
-                # dir_util.copy_tree(entry, destination_file)
-                # shutil.copytree(entry, destination_file, dirs_exist_ok=True)
```

Comment:
I find it astonishing that the module that was causing a problem wasn't even in use from the looks of it.